openapi/components/schemas/Invoices/Invoice.yaml

```
@@ -83,6 +83,9 @@ allOf:
         nullable: true
         type: integer
         readOnly: true
+      organizationId:
+        allOf:
+          - $ref: ../OrganizationId.yaml
       _links:
         type: array
         description: The links related to resource.
```

nitpick: Where possible, we should avoid starting descriptions with "the"

This can be tiresome to read

```suggestion
    description: Related resource links.
```